donkeycar/templates/cfg_complete.py

```diff
@@ -567,19 +567,28 @@
 DONKEY_GYM = False
 DONKEY_SIM_PATH = "path to sim" #"/home/tkramer/projects/sdsandbox/sdsim/build/DonkeySimLinux/donkey_sim.x86_64" when racing on virtual-race-league use "remote", or user "remote" when you want to start the sim manually first.
 DONKEY_GYM_ENV_NAME = "donkey-generated-track-v0" # ("donkey-generated-track-v0"|"donkey-generated-roads-v0"|"donkey-warehouse-v0"|"donkey-avc-sparkfun-v0")
```

Comment:
@DocGarbanzo while we are changing things, should we unify the names used in the myconfig file ? Some are GYM_ other SIM_, other DONKEY_GYM or both, I think it's quite confusing.
I guess changing this would break people's config ? is there a way to update them ?

Reply:
I would say that `SIM` should be used for all data referring to the simulation app and `GYM` should be used for everything interacting with the donkey gym. So the above looks reasonable to me. But we should clearly distinguish between them and correct it where it seems to be mixed up.